nion/instrumentation/AcquisitionPreferences.py

```diff
@@ -118,6 +118,10 @@ def is_customizable(self) -> bool:
     "delay": Schema.prop(Schema.FLOAT),
 }, ControlCustomization)
 
+default_drift_frame_parameters = {
```

Comment:
Adding a `default_drift_frame_parameters: Persistence.PersistentDictType = ...` to this will probably fix the remaining issue. You'll have to `from nion.swift.model import Persistence` somewhere too.

Reply:
The typing in `Schema.py` needs to be changed somewhat; this is only a workaround.